list/PageableList.js

```diff
@@ -315,8 +221,87 @@ define([
 			};
 		}),
 
+		/*jshint maxcomplexity:14*/
+		computeProperties: function (props) {
+			if (this.pageLength > 0) {
+				if ("_busy" in props || "hideOnPageLoad" in props || "autoPaging" in props || "showNoItems" in props) {
+					this._updateListView();
+				}
+
+				if ("loadPreviousMessage" in props || "loadingMessage" in props || "_showPreviousPageLoader" in props) {
+					if (this.previousPageLoader && this.previousPageLoader.deliver) {
+						this.previousPageLoader.labels = {
+							loadMessage: string.substitute(this.loadPreviousMessage, this),
+							loadingMessage: this.loadingMessage
+						};
+						this.previousPageLoader.deliver();
```

Comment:
PS: Having gone that far, you might as well go the extra step and just define a property `this.PreviousPageButtonLabel` (or something like that), and then let the template reference it, like:

```
<d-list-loader label="{{previousPageButtonLabel}}"></d-list-loader>
```

That's in line with how other template properties are set, and then you don't need to manually call `deliver()` anymore.